applications/luci-app-wol/root/usr/libexec/rpcd/luci.wol

```diff
@@ -0,0 +1,90 @@
+#!/bin/sh
```

Comment:
recommend this be written as ucode, then you avoid manual json handling pitfalls, and just build objects directly.